config/routes.rb

```diff
@@ -208,6 +208,10 @@
             get "/:org_username", to: "organizations#show"
           end
         end
+
+        namespace :admin do
+          resources :configs, only: %i[show create], defaults: { format: :json }
```

Comment:
Do we plan to access single keys separately? Otherwise I'd suggest using the [singular resources routes](https://guides.rubyonrails.org/routing.html#singular-resources)

Something like:

```ruby
        namespace :admin do
          resource :configuration, only: %i[show]
        end
```

This will generate the following route:

```
                           api_admin_configuration GET      (/locale/:locale)/api/admin/configuration(.:format)                                    api/v0/admin/configurations#show {:locale=>nil, :format=>"json"}
```

and you can access it with: `curl localhost:3000/api/admin/configuration` (no need for `all` or anything else)

Reply:
Changed to `resource :config` ✅